### PR TITLE
Create petr-baranovskiy-covid-ca-data-explorer.yml

### PR DESCRIPTION
**Pull request by baranovskypd**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/petr-baranovskiy-covid-ca-data-explorer.html
- [x] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

**Comment by riccardoporreca** [comment]
@baranovskypd, thanks for your submission. We have to find a workaround the "mixed content" issue related to your app not being reachable via HTTPS (I did not realized this upon your question by email), as you also point out in your blog post.

This unfortunately prevents the app from being embedded in the [CovidR Gallery](https://milano-r.github.io/erum2020-covidr-contest/), but should not be a hard blocker. My simplest suggestion is to embed a screenshot with an external link to the actual app. The easy way to do so is to create a GitHub [Gist](https://gist.github.com/). I have created a minimal version of this as a PoC at https://gist.github.com/riccardoporreca/c23cbca4e9cc2e7d5d012c696c61ae78.
If you are OK with this solution, I would suggest you to Fork the Gist from the link above and possibly edit it to your best convenience: you may consider adding some text, possibly even along the lines of your [blog post](https://dataenthusiast.ca/?p=1033) (which is where I took the image from).

Sorry for the inconvenience, let us know if you have questions or would like to propose alternatives or discuss more in detail.

**Comment by baranovskypd** [comment]
Hi Riccardo,

Thank you for notifying me about this issue. I did as instructed with the Gist, with the following edits: I replaced the image with a higher-resolution one and added a phrase "Please click the picture, and it will take you to the Shiny app."

I have also uploaded a version of the same app to shinyapps.io. I had to make minimal changes to the code so that R, instead of cron, downloads the data from canada.ca.

The link to my app on shinyapps.io: https://peterbar.shinyapps.io/covid_ca_covidr/

Feel free to choose the option that you believe would work best for the format of the contest. Please let me know which one you decided to go with.

All the best,

Petr

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by riccardoporreca** [comment]
Hi @baranovskypd, thanks for the prompt action. 
I would suggest we go for the shinyapps.io version, which is more _fair_ to you as contributor in terms of visibility. Will go ahead with this and will start integrating the shinyapps.io version soon, let me know if you have further questions.
riccardo

File: contest/contributions/petr-baranovskiy-covid-ca-data-explorer.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  COVID-19 Canada Data Explorer is an interactive Shiny app to break down and analyze
+  the official COVID-19 dataset available from the Government of Canada
+  (https://health-infobase.canada.ca/src/data/covidLive/covid19.csv).
+  The app can be used to visualize 13 different indicators by time and geography, and
+  to directly compare the epidemiological situation across Canada's provinces and
+  territories. It also allows users to custom-build and download their own datasets.
+menu_entry: COVID-19 Canada Data Explorer
+repository: https://github.com/baranovskypd/covid_ca_covidr
+type: shiny
+content: http://dataenthusiast.ca:3838/covid_ca/
```
Comment (riccardoporreca) [inline contest/contributions/petr-baranovskiy-covid-ca-data-explorer.yml]: New HTTPS shinyapps.io URL
```suggestion
content: https://peterbar.shinyapps.io/covid_ca_covidr/
```

File: contest/contributions/petr-baranovskiy-covid-ca-data-explorer.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  COVID-19 Canada Data Explorer is an interactive Shiny app to break down and analyze
+  the official COVID-19 dataset available from the Government of Canada
+  (https://health-infobase.canada.ca/src/data/covidLive/covid19.csv).
+  The app can be used to visualize 13 different indicators by time and geography, and
+  to directly compare the epidemiological situation across Canada's provinces and
+  territories. It also allows users to custom-build and download their own datasets.
+menu_entry: COVID-19 Canada Data Explorer
+repository: https://github.com/baranovskypd/covid_ca_covidr
+type: shiny
+content: https://peterbar.shinyapps.io/covid_ca_covidr/
```
Comment (riccardoporreca) [inline contest/contributions/petr-baranovskiy-covid-ca-data-explorer.yml:13]:
```suggestion
content: https://peterbar.shinyapps.io/covid_ca_covidr/
date: 2020-05-18-15:00
```

**Review by riccardoporreca — APPROVED** [review]

**Comment by riccardoporreca** [comment]
@baranovskypd your submission is now accepted and the shinyapps.io version of the app is visible at https://milano-r.github.io/erum2020-covidr-contest/petr-baranovskiy-covid-ca-data-explorer.html :tada:
If you have have further questions feel free to follow-up in this same pull request.

You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Comment by baranovskypd** [comment]
Thank you for the updates!

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Pull request merged**